api/pkg/transformer/pipeline/upi_preprocess_output_op.go

```diff
@@ -51,7 +52,10 @@ func (up *UPIPreprocessOutputOp) Execute(ctx context.Context, env *Environment)
 
 	transformerInput := &upiv1.TransformerInput{}
 	transformerInput.Tables = transformerInputTables
-	copiedRequest := enrichedRequest
+	copiedRequest := &types.UPIPredictionRequest{}
+	if err := copier.CopyWithOption(copiedRequest, enrichedRequest, copier.Option{IgnoreEmpty: true, DeepCopy: true}); err != nil {
```

Comment:
Why do we copy? We should minimize amount of copy to minimze the overhead introduced by standard transformer.